### PR TITLE
SM-6075 - expanding docs with new forward plan progression rules

### PR DESCRIPTION
**Pull request by cpaul92**

### CODE REVIEW

**Review by KylerD — APPROVED** [review]

File: docs/api-documentation/V1.25.4/index.md [hunk 1]
```diff
@@ -1947,19 +1947,21 @@ permit_asds for the provided USRN can be found at Street Lookup API endpoint /ns
...
 
-##### Progress a forward plan to a PAA
+##### Progress a forward plan to a PAA or Permit
+{: .govuk-heading-s}
 
-When a forward plan is created, a works is created with no permits. In this scenario, this endpoint is used to raise the first permit on the works. This permit will always be a PAA. This endpoint requires some of the same fields as the create forward plan request but much of the information from the forward plan will be used as the value for the PAA, and so only a subset of information is required.
+When a forward plan is created, a works is created with no permits. In this scenario, this endpoint is used to raise the first permit on the works. This permit will be a PAA or Permit depending on the type of work and the duration. A forward plan with minor or standard duration, or any highway works will progress straight to a permit. A forward plan will progress to a PAA if the duration falls under the major category or if ttro is required. This endpoint requires some of the same fields as the create forward plan request but much of the information from the forward plan will be used as the value for the PAA, and so only a subset of information is required.
```
Comment (caoimheb) [inline docs/api-documentation/V1.25.4/index.md:1961]:
```suggestion
When a forward plan is created, a works is created with no permits. In this scenario, this endpoint is used to raise the first permit on the works. This permit will be a PAA or Permit depending on the type of work and the duration. A forward plan with minor or standard duration, or any highway works will progress straight to a permit. A forward plan will progress to a PAA if the duration falls under the major category or if ttro is required. This endpoint requires some of the same fields as the create forward plan request but much of the information from the forward plan will be used as the value for the PAA, and so only a subset of information is required.
```
```suggestion
When a forward plan is created, a works is created with no permits. In this scenario, this endpoint is used to raise the first permit on the works. This permit will be a PAA or Permit depending on the type of work and the duration. A forward plan with minor or standard duration, or any highway works will progress straight to a permit. A forward plan will progress to a PAA if the duration falls under the major category or if ttro is required. This endpoint requires some of the same fields as the create forward plan request but much of the information from the forward plan will be used as the value for the PAA/permit, and so only a subset of information is required.
```

Comment (caoimheb) [inline docs/api-documentation/V1.25.4/index.md:1961]:
```suggestion
When a forward plan is created, a works is created with no permits. In this scenario, this endpoint is used to raise the first permit on the works. This permit will be a PAA or Permit depending on the type of work and the duration. A forward plan with minor or standard duration, or any highway works will progress straight to a permit. A forward plan will progress to a PAA if the duration falls under the major category or if ttro is required. This endpoint requires some of the same fields as the create forward plan request but much of the information from the forward plan will be used as the value for the PAA/permit, and so only a subset of information is required.
```

Comment (caoimheb) [reply docs/api-documentation/V1.25.4/index.md:1961]: Diff doesn't show it very clearly, but update to the last sentence to imply forward plan can go to either PAA or permit

Comment (cpaul92) [reply docs/api-documentation/V1.25.4/index.md]: Updated

**Review by caoimheb — APPROVED** [review]

**Pull request merged**